talktorials/11_online_apis/11c_quality_assessment.ipynb

```diff
@@ -0,0 +1,1370 @@
omPhImGuutNM9jjZUMkxaKb6b06+2ik0zzkfBZk9hspPZpl5KUcbxEpOwQ+g4SRSUBLSgBAA4STUgMMPYAWYeAZ4ZhGg4M4a4B9i/lIB/iFGTkEHUEIFXMBGJDtX+8cvSmURb2cbAICcb0QJAoEPgqgp2OIU2OAJLrnDJ31mhuBH4q1f82iUN/3FAeyw/DR/oVBB8Uao44JvzoRRH1bSMBZRaVOSbHMMBzVDP8whpHq1RjQcW2p0RfbjPq3JSZQqbwLKV8J1TXV2R6aYqxxkSidmOw+5MIRDTmEMpPgkqu4JYEC+L0QW2IAN7sgvnooTJKGMJV1Utqojz+oRNEEC1J0lj6wP3ICPdy0aRGALEuAkGUATmsBNH/0O8pLS//12soVfdfHE2UWQfvqT1yWQFNfgB7ZAUSNhBW5Rk3EiKP6VyTAZFdXC+EXi/lKdgCt2IyiuuXk9ZKl6RD2Jg34dg78xdFaWZjkPArN8qEp4rF93Rl/uZkcEm6n9JLoGANobQxAAAOzAoxkAAOr7dmywI/snFzRBAOwA6qFejW2uD56KAZ4+6u3ArT3yFFahEW555vjiFMgd4IWotTNjeks8jwhUp4E6NCErGHn6M4P6w98iCzG0Qw1HuOeFuIPDUSmJwyZJTI6ybxKCUjlt5Tc4HH+dyTNu5WcL44adakJYrGuehI0rdq+idpUddyHkE1zBDVLiE6ZpaEckk2FAEyBgAYqoBQR6ZqrnsRvEL/pgEghBpnltCVZBEQS77IFsv+Y3oErTsY4EcQeW/t/uDflD3TR+Uv0ySjdRfcNo3cbtJU00iUseqcIwfnFcnSQgmMhfTFNFbUVDVDhFtjgBhQ3BUVTxgphptNmnXNmxXIQpMOcxpA8gwQ5Qgi3+wQ9Km0QUACAIJANAkA2NXAcSKlzIsKHDhxAjOnTgao1Eh7lyKeh3kWEfSG4QLupIsqTJkyUbLFFiQAHKlzBjypxJs6bNh0OgAbjg0mEiffcsWdInlKjQoUWRGk1K1OhSpEqhOpWatGrVqVGvUlWq75KUl7qqxBNLT6zZsfHGViG7Fu3ZtWzTmm0rN61bt2bL3tWL1u5ZvnDj6T2L9svIl9HcsVvMjloMxoypQZ5M/rkyu8eNMVN2zFiz5cudJ3OurFlzJo4oo6lCpq6169ewY2NqPTu27duxM+Dezdt17drIDGX6erPjKYQMT/UpblIWIiURkDOfntAEACgoT6WDpOifSY593LhZzo+6+fPo06tfz55mLlQA4MhyGC3RvXtYm0Zdql9r0/xHYbUfUwNOBaB+BjJVCXEn6WKBXxDOJZiEYwEGmFx/SdhWFYNNeNeHb8UlFlodVijhF87AlNhnjYEGmWQturhYaSxG5lmNLMI4GWY3QpYJTPyQwJpsren22m+7GVmbOkjatuRrRrqmpG+0HUklbsHF4E97XHbpZQ8VJbRlSny4MEgELzXg/gYMyx3m5ZtwxinnnNPlAgsAqMzXUD/2EYigfwVudSBTAvaXVX+IbkUVfwuW1+CGapmYVol1aVgpiW95WCJhEVba4YiXggppXxaMmZpikvV4Y6oz4ljjaDXSuBmPrk72hopCWulkb0zyWuWvukLpK25PFqmOcHQmq2xJVriyQpy5uCHHJwc4uix6ueARSBg9Xevtt9DqdEEoD/0UVKFaLRqgoIAmqC5/ihqa4FP6VXIrSg5i6GGIlRImIqZ94VXXpWpJOFiHg20IWKZ5VWGYijpC1mOrn61KsaykhRaZxJlVbBmt7Pz4Ej+qDGsybMXyFuWVvq58W5RZmgruzHIi/uDsnA00IgeaqNF8UyDJFMKFEz4XbTRzQ8AHBy302RfUu+nSCxXUg/q57p8DHmoV1vo0iu+InA4s8MH+UhrYhQoTtvBdYNf1KYkAF+bmSSvWmuOOFNc6sd2fwRrxYiKnluttKQOrjsu9Ji4lbIijTCywhRfuWnCBH205eggAYFFMfdjQiR2BwMRRP5DI4cPlNSngjHSot+46RLnopAG5e9rH9brszstu7lrDG6+6gRoYQ88m6YKP2xtymOmFdvUl6oibMgywvwGDHXbD1t9Vqooaewbj3iDHyLeMol3scd5/V3YvSlKsFqzjRC7Oa+GNx98b/bYJF0WyMnc5pj/E/vNWFDxAkCdgISFKsANCGkASLNRAAAIgmkl68IgD7M8EXAAAJE73ug568IMysdNOWLeQPunuTyicmu6yliiqAWpru5NKV6xVPH45Ty300JfA1oK2fV1KL2hrGxA1RBYMtWVTftHLF/pXksSkT2N4O1/5XBS+F1GxMrDaG/lAJpnKmSQag3vcyWxTvyctSXJjtF8GsgRCmziKht46hRkAIAkAEOKAp3ADIUjInP39owubowNB6piANhrykIhUSC7ggCc9MaQ+AIoa7nbnO3ilEHi8u90Ji7IgsDyoej4EkaSyF6K2Kc95mSIYXeKyl+zJ5W3xeBhiFDOrjFmxb3nD/tETN6NLLYbGiyUJ0pDgZz+WJQ6NxuQN/s5IxmS2RjhMJIkocEHNalrzmtjMpja3yc1uevOb1jxFlwIxxz3YUZy5IIQbXBLAmkTjAP9whQsSIgKCmFMPfEzkQk6hhxrUAE36DKiy3rOTbi2ET/dJlLtSeElNpitrwSPQ79q1IAaVJCwNY5sPM8rKV4LKLdFDXtxEVRbq0aV5kOKULE+1GFZ9TDSe8Z4WYRSx71nGplKkzN6ASRKSEVNlvjLj/HaDzPdJjo0vOcUr7MHUpjr1qVCNqlSnStWqWvWqTi2Gl5TABjoCwA3iXEQdIEEuONbEC/JMSOa8WgOOmFWgEeiE/j85KNC6QutOF3AkQ35yFODdDoYPbeFCrebXTGYlKfYCi/Rs+C9TRoqjOTSpKJNXsOqZLTBlO8tK6YbTWt4Sii6NoqxgpbHQcqxjuHyRaXA1TPsVVXH3I9xPX2s43lAumhdRBjrsMY7e2kMc4vhtcJsxDnsQVxzjAK5yl8vc5SbXuciNbm+R
```

Comment:
FYI: In case, you intended this to look differently: These three points you are making here, starting with `>`, do not show up in new lines but as a continuous text separated by `>>`. Though, to me, It is fine to read as-is.

Reply:
I think you are commenting on the wrong line, because I can only see the embedded image here. Which text are you referring to?

Reply:
Nvm it's a GitHub bug.

Reply:
This is how it renders to me:

![image](https://user-images.githubusercontent.com/2559438/64718730-1658a780-d4c7-11e9-934e-cd7b5da4db47.png)

Reply:
Ok, looks the same in my Jupyter notebook. Maybe I did look at it on GitHub, sorry for the confusion.